src/widgets/SearchBox.js

```diff
@@ -0,0 +1,16 @@
+import { searchBox as searchBoxWidget } from 'instantsearch.js/es/widgets';
+
+export const searchBox = searchBoxWidget({
+  container: '[data-widget="searchbox"]',
```

Comment:
these containers with data attributes are interesting, but the reason we normally use ids, is that there can be only one on the page. Not sure that matters here, since it's an example, but I can imagine someone making multiple refinement lists, and calling them the same, then being confused. What do you think?